### ISSUE TITLE
Onvolkomenheden in aanhaling landelijke lijst et Design Rules en Principles

### ISSUE THREAD

**Issue by melsk-r** [VNG-Realisatie/API-Kennisbank#72]
In deze Kennisbank staan in [de aangehaalde landelijke lijst met Design Rules en Principles](https://github.com/VNG-Realisatie/API-Kennisbank/tree/master/Design%20rules) staan een aantal onvolkomenheden.

1. Ten eerste worden er in de Principles en Best Practices items genoemd die al bij de Design Rules staan. Dit geldt voor alle genoemde Principles en mogelijk voor alle genoemde Best Practices (omdat de 2e en 3e wellicht gelijk zijn aan resp. de Design Rules API-19 en API-20).
2. Ten tweede mis ik in de lijst op deze kennisbank diverse Design Rules (API-53 t/m API-57) en Principles (API-12 t/m API-15, API-21 t/m API-47, API-49, API-50 en API-52).
3. API-09 wordt genoemd maar bestaat volgens mij helemaal niet meer op https://publicatie.centrumvoorstandaarden.nl/api/adr/. Klopt dat?
4. API-05 heeft in deze kennisbank de titel 'Use plural nouns to indicate resources' in de landelijke API strategie echter 'Use nouns to name resources'. Het gebruik van meervoud in endpoints lijkt dus geen wet van Meden en Perzen meer of zie ik dat verkeerd? Alleen als er sprake is van collections moet je meervoudige namen gebruiken maar dat is een andere Design Rule (API-54). **Betekent dit nog wat voor onze API's?**
5. Tenslotte zie ik dat de in deze kennisbank gebruikte titels voor de Design Rules afwijken van de op https://publicatie.centrumvoorstandaarden.nl/api/adr/ gebruikte titels. Was er een reden af te wijken van de naamgeving daar?

Ik wil deze aanpassing met alle plezier even uitvoeren maar wil dan graag op de bovengenoemde punten 3, 4 en 5 een antwoord hebben.

**Comment by JohanBoer** [comment]
> 3. API-09 wordt genoemd maar bestaat volgens mij helemaal niet meer op https://publicatie.centrumvoorstandaarden.nl/api/adr/. Klopt dat?

Klopt. Blijkbaar is die van het normatieve deel naar de Extensions verplaatst. 

> 4. API-05 heeft in deze kennisbank de titel 'Use plural nouns to indicate resources' in de landelijke API strategie echter 'Use nouns to name resources'. Het gebruik van meervoud in endpoints lijkt dus geen wet van Meden en Perzen meer of zie ik dat verkeerd? Alleen als er sprake is van collections moet je meervoudige namen gebruiken maar dat is een andere Design Rule (API-54). **Betekent dit nog wat voor onze API's?** 

Even aanpassen. De bedoeling is echt een verwijzing naar de bestaande design rules, maar blijkbaar wijzigen ook de normatieve design rules gaandeweg.  Overigens heeft dat verder geen impact op onze huidige specificaties. Wij gebruikten al enkelvoud waar we dat nodig vonden. 

> 5. Tenslotte zie ik dat de in deze kennisbank gebruikte titels voor de Design Rules afwijken van de op https://publicatie.centrumvoorstandaarden.nl/api/adr/ gebruikte titels. Was er een reden af te wijken van de naamgeving daar?

Daar zit geen diepere gedachte achter. Misschien heb ik de werkversie gebruikt waar andere titels in staan of zijn deze titels gewijzigd in de tussentijd.
